app/src/main/java/com/github/se/travelpouch/model/travels/TravelRepositoryFirestore.kt

```diff
@@ -175,7 +174,7 @@ class TravelRepositoryFirestore(private val db: FirebaseFirestore) : TravelRepos
 
     Log.d("TravelRepositoryFirestore", "getTravels")
     db.collection(collectionPath)
-        .whereArrayContains("listParticipant", profileUid)
+        .whereArrayContains("listParticipant", CurrentProfile.currentProfile.fsUid)
```

Comment:
You need to change this with the ProfileViewModel and don't use CurrentProfile anymore